vsutil/__init__.py

```diff
@@ -251,6 +251,65 @@ def is_image(filename: str, /) -> bool:
     return types_map.get(path.splitext(filename)[-1], '').startswith('image/')
 
 
+def scale(value: float, 
```

Comment:
This code doesn't even have dependencies on vapoursynth. Why not add a test?

Reply:
I made the test, and I think it's thorough, but I have no idea how to run it.

Reply:
Alright, everything looks good on that end, unless I missed a parameter combination somehow.